content/en/blog/_posts/2021-01-28-PSP-deprecation.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog
+title: 'What’s Up with Pod Security Policies?'
+date: 2021-02-04
+slug: psp-update-2021
+---
+
+**Authors:** Contributor Comms Team
+
+_The following is a brief overview of upcoming changes to Kubernetes. Bookmark and [keep an eye on our release site](https://www.kubernetes.dev/resources/release/) to see the latest news and other important updates._
+Pod Security Policies (PSP), starting with the Kubernetes 1.21, will begin the [process of deprecation](/docs/reference/using-api/deprecation-policy/) with the intention to fully remove it in a future release. This prompts the question: what will replace Pod Security Policies in the future? In short, [Admission Controllers](/docs/reference/access-authn-authz/admission-controllers/). For more details, keep reading.
+
+
+## What do Pod Security Policies do now?
+
+> A Pod Security Policy is a cluster-level resource that controls security sensitive aspects of the pod specification. The [PodSecurityPolicy](/docs/reference/kubernetes-api/policies-resources/pod-security-policy-v1beta1/) object defines a set of conditions that a pod must run with in order to be accepted into the system, as well as defaults for the related fields. They allow an administrator to control a number of security-related functions.
+
+The [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+documentation explains this in more detail.
+
+
+## Why are Pod Security Policies being deprecated? 
+
+To watch an explanation of PSP removal, you can check out this session by SIG-Auth at KubeCon NA 2019:
+
+{{< youtube "SFtHRmPuhEw?start=953" youtube-quote-sm >}}
+
+In short, PSPs are part of the kubelet and API server. The tight coupling to these parts of Kubernetes makes PSPs cumbersome both for developers and for operators. Using PSPs is often unintuitive, and setting them up too often leads to unintended results. The initial design of PSPs did not foresee many of the use cases they’ve since tried to solve.
+
+To quote the above video: "Various limitations and structural problems have prevented the PSP API from GA." Having objects in Kubernetes never reaching GA (“permabeta”) is a problem for administrators and organizations, as a non-GA feature is not considered to be ready for widespread use. Given the structural problems with the PSPs as they exist today, there is no real path for them to become GA. Per the deprecation issue, “the future of restricting pod security settings does not lie in PSP because compatibility restrictions will prevent the kinds of changes that are required.” As such, PSPs will be deprecated in Kubernetes 1.21, though they will not be fully removed until version 1.25.
+
+
+## What will replace them? 
```

Comment:
When is this targeted for publication? We are still hoping to have agreed upon a replacement design prior to v1.21. Even if we haven't selected a replacement design, I think it's worth mentioning that there is a sig-auth breakout group working towards a replacement.

Reply:
I don't think a publication date is set yet.

Reply:
This should've gone out weeks ago when people started asking me.

Reply:
Personally, I'm glad this post didn't go out weeks ago, because SIGs Auth and Security have been making good progress on a solution that was not reflected in this post. At this time, we are planning on closing this PR, rewriting a post to more accurately reflect current events, and putting in a new one. We feel that a clearer post with more definitive solutions will provide better value for users overall than sooner publication of a post without those things would have. Thanks! See you in that PR :)